tests/lib/rules/no-extra-boolean-cast.js

```diff
@@ -78,17 +84,66 @@ ruleTester.run("no-extra-boolean-cast", rule, {
             }]
         },
         {
-            code: "Boolean(!!foo)",
```

Comment:
The change here was because the original file includes two identical test cases for `Boolean(!!foo)`.